packages/ui/src/components/Icon/__stories__/Color.stories.tsx

```diff
@@ -3,11 +3,7 @@ import type { ComponentProps } from 'react'
 import { Icon } from '..'
 import { Stack } from '../../Stack'
 
-const colors: ComponentProps<typeof Icon>['color'][] = [
-  'primary',
-  'success',
-  'warning',
-]
+const colors = ['primary', 'success', 'warning'] as const
```

Comment:
Constants should be in uppercase:

```suggestion
const COLORS = ['primary', 'success', 'warning'] as const
```